src/components/SubsidyRequestManagementTable/data/hooks.js

```diff
@@ -112,8 +112,38 @@ export const useSubsidyRequests = (
     [],
   );
 
+  const updateRequestStatus = ({
```

Comment:
Opted for this approach instead of reloading requests since it takes time for requests to go from pending -> approved/error, so reloading each time won't get us an updated status.